website/scripts/python-doc-gen.sh

```diff
@@ -5,8 +5,8 @@ HERON_ROOT_DIR=$(git rev-parse --show-toplevel)
 INPUT=heronpy
 TMP_DIR=$(mktemp -d)
 
-sudo pip install heronpy==${HERONPY_VERSION}
-sudo pip install --ignore-installed six
+sudo -H pip install heronpy==0.17.1
```

Comment:
@lucperkins - is this change ok?

Reply:
I will back this change out.  I had to do some tricky stuff to get the pages to build on my local.  I meant to revert this, it slipped my mind.  Good catch.  Thank you